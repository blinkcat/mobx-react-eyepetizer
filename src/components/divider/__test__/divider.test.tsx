import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Divider from '../';

describe('Component Divider', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Divider />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders with short prop correctly', () => {
    const tree = renderer.create(<Divider short />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
