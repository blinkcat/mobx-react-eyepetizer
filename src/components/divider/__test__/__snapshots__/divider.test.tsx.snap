// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Divider renders correctly 1`] = `
<div
  className="divider"
>
  <style
    jsx={true}
  >
    
        .divider {
          background: #fff;
          height: 1px;
          @media (-webkit-min-device-pixel-ratio: 2) {
            transform: scaleY(0.5);
          }
          &.short {
            width: 40px;
          }
        }
      
  </style>
</div>
`;

exports[`Component Divider renders with short prop correctly 1`] = `
<div
  className="divider short"
>
  <style
    jsx={true}
  >
    
        .divider {
          background: #fff;
          height: 1px;
          @media (-webkit-min-device-pixel-ratio: 2) {
            transform: scaleY(0.5);
          }
          &.short {
            width: 40px;
          }
        }
      
  </style>
</div>
`;
